functional-tests/bdd-tests/CRS_request_store_name_branding_change.feature

```diff
@@ -0,0 +1,15 @@
+﻿Feature: CRS_request_store_name_branding_change
+    As a logged in business user
+    I would like to request a Cannabis Retail Store name or branding change
+
+Scenario: Request CRS Store Name or Branding Change
+    Given the CRS application has been approved
+    And I am logged in to the dashboard as a private corporation
+    And I click on the Licences tab
+    And the licence fee has been paid
+    And I click on the Request Store Name or Branding Change link
+    And I review the account profile
+    And I submit a name or branding change application
```

Comment:
Happy path would be "I enter a valid name change"; there are invalid name changes that are handled by the front end. The application is submitted and after a manual review process, the name of the establishment is updated.